Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -635,7 +635,11 @@ private async Task<bool> CanUpdateAsync(string resource)
 
                     return false;
                 }
-
+            case Resource.PROFILE_PRIVACY:
```

Comment:
The new privacy settings now are available to all users, but the call to `StateYourBusiness()` in the ProfileController is commented out and that is the only place this resource is used.  One way to address this would be to uncomment the StateYourBusiness() reference but have it just return true for all users rather than returning true only for facstaff.

Reply:
Now the profile controller does call StateYourBusiness() with the PROFILE_PRIVACY resource.  SYB() always returns true, so behavior is not changed but this makes it easier to add some restrictions if needed in the future.